flink-core/src/main/java/org/apache/flink/configuration/ResourceManagerOptions.java

```diff
@@ -55,6 +55,18 @@
 			" default, the port of the JobManager, because the same ActorSystem is used." +
 			" Its not possible to use this configuration key to define port ranges.");
 
+	/**
+	 * Defines the maximum number of workers (YARN / Mesos) failure can happen in a minute.
```

Comment:
From our production experience, it would be better to let the job failure immediately to make engineer to involve into the issue. The failure rate here is to just distinguish whether it is a transient failure or real issue caused by external systems such as hdfs. Thus, I am not sure it is needed for wait for failure rate falls below the  maximum.